### PR TITLE
AML-2261 AML-2340 AML-2341 Payments import refactor

### PR DESCRIPTION
**Pull request by elliotritchie**

### CODE REVIEW

**Comment by elliotritchie** [comment]
Also included as part of these changes: https://github.com/SkillsFundingAgency/das-shared-packages/pull/217.

File: src/AzureTopicConfigurationStrucuture.json [hunk 1]
```diff
@@ -68,9 +68,5 @@
   {
     "topicName": "user_joined",
     "subscription": [ "Activity_UserJoinedMessageProcessor" ]
-  },
-  {
-    "topicName": "payment_created",
-    "subscription": [ "Activity_PaymentCreatedMessageProcessor" ]
```
Comment (MichaelYoung1981) [inline src/AzureTopicConfigurationStrucuture.json]: If we remove this topic and, as unlikely as it is, build a new environment the topic won't exist and the publication of messages will throw an exception.

File: src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/DasLevyRepository.cs [hunk 2]
```diff
@@ -251,7 +252,7 @@ public DasLevyRepository(LevyDeclarationProviderConfiguration configuration, ILo
             return result;
         }
 
-        public async Task CreatePaymentData(IEnumerable<PaymentDetails> payments)
+        public async Task CreatePayments(IEnumerable<PaymentDetails> payments)
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/DasLevyRepository.cs]: This method could do with a little bit of love.  Maybe extract out a few private methods?

Comment (elliotritchie) [reply src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/DasLevyRepository.cs]: Done

File: src/AzureTopicConfigurationStrucuture.json (hunk 1 above)
Comment (elliotritchie) [reply src/AzureTopicConfigurationStrucuture.json]: Removing the config will prevent the subscription from being created if devops delete it. We can leave the topic there though. That should work?

**Pull request merged**